### ISSUE TITLE
README not rendered correctly on PyPI 

### ISSUE THREAD

**Issue by benjamin-work** [dnouri/skorch#129]
This is apparently caused by the section headings :-1:

**Comment by benjamin-work** [comment]
Since the README is unfortunately still not rendered correctly on PyPI, I re-opened this issue.

I see two ways forward:

* A pypi-rst wizard solves the remaining issues with our README.rst
* We revert the README back to markdown, which supposedly is [now supported on pypi](https://stackoverflow.com/a/26737258).

**Comment by thomasjpfan** [comment]
I ran the following to deploy onto https://test.pypi.org.

```bash
python setup.py sdist
twine upload --repository testpypi dist/*
```

This properly renders most of the README: https://test.pypi.org/project/skorch-doc-test/. How are we pushing releases to pypi?

**Comment by benjamin-work** [comment]
Very curious, I'm certain we do it the same, @ottonemo you see any difference?

**Comment by ottonemo** [comment]
Currently we publish like this:

```Makefile
dist:
	python setup.py sdist bdist_wheel

publish: dist
	twine upload --repository-url https://test.pypi.org/legacy/ dist/*$$(cat VERSION)*
```

**Comment by hsiaoyi0504** [comment]
It seems to me that PyPA has a good document about this: https://packaging.python.org/guides/making-a-pypi-friendly-readme/.

**Comment by hsiaoyi0504** [comment]
I just use new feature of twine checking the rst and it looks good
https://packaging.python.org/guides/making-a-pypi-friendly-readme/#validating-restructuredtext-markup.

``` shell
> python setup.py sdist bdist_wheel
> twine check dist/*
Checking distribution dist/skorch-0.4.0a0-py2-none-any.whl: Passed
Checking distribution dist/skorch-0.4.0a0.tar.gz: Passed
```

**Comment by BenjaminBossan** [comment]
Thanks for the link. In theory, our README.rst should be fine, we are not sure why it doesn't render on PyPI. Perhaps the best course of action is to revert to markdown, which should now be supported by PyPI and is sufficient for our README.

**Comment by thomasjpfan** [comment]
This issue should be resolved with #380. We can confirm this by pushing a release to the testing pypi server: https://test.pypi.org/project/skorch/

**Comment by benjamin-work** [comment]
This issue is indeed resolved (thank god).
